common/changes/sparo/feat-sparo-branch_2024-05-21-23-13.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "\"sparo branch -d\" clean up remote fetch git configuration",
```

```suggestion
      "comment": "\"sparo branch -d\" clean up remote fetch git configuration",
```
This sentence is not understandable to someone unfamiliar with your PR.  [Review these docs](https://rushjs.io/pages/best_practices/change_logs/) for tips about authoring change messages.